index.html

```diff
@@ -110,14 +110,74 @@ <h3>let's connect</h3>
           <li class="expertise">
             <h5>Languages</h5><img src="./logos/Union.png" alt="expand" class="arrow"></li>
             <li><ul id="languages-list">
-              <li class="languages"><h6>JavaScipt</h6><img src="./logos/Ellipse1.png" alt="javascript logo"></li>            
-              <li class="languages"><h6>HTML</h6><img src="./logos/Ellipse2.png" alt="html logo"></li>            
+              <li class="languages"><h6>JavaScipt</h6><img src="./logos/Ellipse1.png" alt="javascript logo"></li>
+              <li class="languages"><h6>HTML</h6><img src="./logos/Ellipse2.png" alt="html logo"></li>
               <li class="languages"><h6>CSS</h6><img src="./logos/Ellipse3.png" alt="css logo"></li>
             </ul></li>
           <li class="expertise"><h5>Frameworks</h5><img src="./logos/Union2.png" alt="expand" class="arrow"></li>
           <li class="expertise"><h5>Skills</h5><img src="./logos/Union2.png" alt="expand" class="arrow"></li>
         </ul>
       </section>
+
+    <section id="contact-section">
+      <h1>Contact me</h1>
+      <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      <form action="https://formspree.io/f/mzbokdyd"  method="post" id="contact-form">
+      <fieldset>
+        <input type="text" id="name" placeholder="Full Name" maxlength="30" required>
+        <input type="email" id="email" placeholder="Email Address" required>
+        <textarea name="message" id="message" cols="30" rows="4" maxlength="500" required>Enter Text Here</textarea>
```

Comment:
- [ ] [OPTIONAL] You can use `placeholder` for `Enter Text Here` instead of making it content on the page, it's better for the user.